def/ru/menu_darwin.cson

```diff
@@ -3,317 +3,323 @@ Menu:
     value: "Atom"
     submenu:
       "About Atom":
-        value: "About Atom"
+        value: "О Atom"
       "View License":
-        value: "View License"
+        value: "Лицензия"
+      "Check For Update":
+        value: "Проверить обновления"
```

Comment:
Thanks for pointing out new item 

`"Check For Update"` to be translated. 

Confirmed this shows up in win32 platform also (not in linux).
May you help to add this to `menu_win32.cson` also? :smile: